From: thomas@winischhofer.net
Subject: [PATCH] USB: SiS USB2VGA minor fix.
ChangeSet 1.2020.1.3, 2005/03/07 22:16:51-08:00, thomas@winischhofer.net

[PATCH] USB: SiS USB2VGA minor fix.

here is a tiny, small update for the sisusb driver. It fixes one spacing
issue, one (internal) API issue and one endian issue (stupid copy/paste
error)

Signed-off by: Thomas Winischhofer <thomas@winischhofer.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 drivers/usb/misc/sisusbvga/sisusb.c |    3 ++-
 drivers/usb/misc/sisusbvga/sisusb.h |    4 ++--
 2 files changed, 4 insertions(+), 3 deletions(-)


diff -Nru a/drivers/usb/misc/sisusbvga/sisusb.c b/drivers/usb/misc/sisusbvga/sisusb.c
--- a/drivers/usb/misc/sisusbvga/sisusb.c	2005-03-08 16:56:39 -08:00
+++ b/drivers/usb/misc/sisusbvga/sisusb.c	2005-03-08 16:56:39 -08:00
@@ -1813,7 +1813,7 @@
 	SETIREG(SISSR, 0x02, 0x0f);
 	SETIREG(SISSR, 0x03, 0x00);
 	SETIREG(SISSR, 0x04, 0x0e);
-	SETREG(SISMISCW,0x23);		/* misc */
+	SETREG(SISMISCW, 0x23);		/* misc */
 	for (i = 0; i <= 0x18; i++) {	/* crtc */
 		SETIREG(SISCR, i, crtcrdata[i]);
 	}
@@ -2833,6 +2833,7 @@
 			x.sisusb_pcibase    = SISUSB_PCI_PSEUDO_PCIBASE;
 			x.sisusb_vramsize   = sisusb->vramsize;
 			x.sisusb_minor	    = sisusb->minor;
+			x.sisusb_fbdevactive= 0;
 
 			if (copy_to_user((void __user *)arg, &x, sizeof(x)))
 				retval = -EFAULT;
diff -Nru a/drivers/usb/misc/sisusbvga/sisusb.h b/drivers/usb/misc/sisusbvga/sisusb.h
--- a/drivers/usb/misc/sisusbvga/sisusb.h	2005-03-08 16:56:39 -08:00
+++ b/drivers/usb/misc/sisusbvga/sisusb.h	2005-03-08 16:56:39 -08:00
@@ -50,7 +50,7 @@
 
 #define SISUSB_VERSION		0
 #define SISUSB_REVISION 	0
-#define SISUSB_PATCHLEVEL	6
+#define SISUSB_PATCHLEVEL	7
 
 /* USB related */
 
@@ -83,7 +83,7 @@
 	do {						\
 		p->header  = cpu_to_le16(p->header);	\
 		p->address = cpu_to_le32(p->address);	\
-		p->address = cpu_to_le32(p->data);	\
+		p->data    = cpu_to_le32(p->data);	\
 	} while(0)
 #else
 #define SISUSB_CORRECT_ENDIANNESS_PACKET(p)
